changelog_unreleased/typescript/11515.md

```diff
@@ -1,6 +1,7 @@
-#### Avoid extra offset in arrow function body when using long types (#11515 by @kachkaev and @thorn0)
+#### [HIGHLIGHT]Avoid extra offset in arrow function body when using long types (#11515 by @kachkaev and @thorn0)
 
 Starting with Prettier 2.3.0, type declarations in arrow functions could affect function body offset. Changing the length of the type annotation could produce large diffs and thus increased the chance of git conflicts. To prevent this, function body offset was stabilized.
+**Note**: This change may affect a large number of lines in your codebase.
```

Comment:
```suggestion
Starting with Prettier 2.3.0, type declarations in arrow functions could affect function body offset. Changing the length of the type annotation could produce large diffs and thus increased the chance of git conflicts. To prevent this, function body offset was stabilized.<br>
**Note**: This change may affect a large number of lines in your codebase.
```